### ISSUE TITLE
Default set to None doesn't work

### ISSUE THREAD

**Issue by asxlvm** [Arthurdw/dogehouse.py#5]
Hey so, I'm not sure if this is my problem but;
```
  @command
  async def covid(self, ctx: Message, *, country = None):
    if country == None:
      await self.send("You have to specify a country to see their COVID stats! Example: d!covid Czechia")
      return
    else:
      pass
    
    cases = Covid().get_status_by_country_name(country.lower())
    
    region = cases["country"]
    confirmed = cases["confirmed"]
    active = cases["active"]
    deaths = cases["deaths"]
    recovered = cases["recovered"]
    
    await self.send(f"COVID stats for {region} • Confirmed Cases: {confirmed} • Active Cases: {active} • Deaths: {deaths} • Recovered: {recovered}")
```
This is my command, it works normally but when the user doesn't specify the country, it should send the message "You have to specify a country..." but it doesn't, this works on Discord.py, not sure why it doesn't here! :D

**Comment by Arthurdw** [comment]
Currently an error gets thrown when not enough arguments are supplied. (Which is recommended to be handled in the `on_error` event listener)

But I'll take a look tomorrow on how I can fix this.

(& for your example, you can check the `ctx.content` for the country, just so you can work further on your bot. ^^)

**Comment by asxlvm** [comment]
I don't wanna make another issue on this but;

When I use the author of the message it doesn't work, example:
```
@command
async def test(self, ctx: Message):
    await self.send(f"{ctx.author.mention}"}
```

Doesn't return anything, not even an error, am I making a mistake?

**Comment by Arthurdw** [comment]
That shouldn't happen, I'll take a look.

**Comment by Arthurdw** [comment]
> I don't wanna make another issue on this but;
> 
> When I use the author of the message it doesn't work, example:
> 
> ```
> @command
> async def test(self, ctx: Message):
>     await self.send(f"{ctx.author.mention}"}
> ```
> 
> Doesn't return anything, not even an error, am I making a mistake?

You didn't close the function bracket properly (You used the wrong bracket type).
So the fix would be:
> ```py
> @command
> async def test(self, ctx: Message):
>     await self.send(f"{ctx.author.mention}")
> ```

**Comment by asxlvm** [comment]
Oh, I was sleepy then 😆
I'll come back if I get any more errors

**Comment by Arthurdw** [comment]
The feature has been implemented in `0.6.0`
